baremetal/metal3remediation_manager_test.go

```diff
@@ -334,7 +335,7 @@ var _ = Describe("Metal3Remediation manager", func() {
 			},
 			ExpectPresent: false,
 		}),
-		Entry("Should not find the host, annotation not present", testCaseGetUnhealthyHost{
+		Entry("Should not find the host, host annotation is empty", testCaseGetUnhealthyHost{
```

Comment:
It is not host annotation but rather m3m if you wanna specify, otherwise should be fine as is